scripts/build_shapes.py

```diff
@@ -36,8 +37,7 @@
 from shapely.validation import make_valid
 from tqdm import tqdm
 
-logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+sys.excepthook = handle_exception
```

Comment:
This shall be placed into the create_logger fuction but here we shall create the local logger like:
`logger = create_logger(__name__, logging.INFO)`